vignettes/articles/tbl_ard-functions.Rmd

```diff
@@ -0,0 +1,125 @@
+---
+title: "Building Tables using tbl_ard*()"
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+For most tables built using {gtsummary}, ARDs for quality control or table summaries can be extracted. However, there are instances where an ARD first approach is beneficial as building more complex tables may require analysis of the data structures prior to table construction. For this ARD first approach, {gtsummary} has `tbl_ard*()` functions that will generate a tables.
+
+-   `tbl_ard_continuous()` - for ARDs summarizing continuous variables.
+-   `tbl_ard_hierarchical()` - for ARDs containing nested or hierarchical data structures.
+-   `tbl_ard_summary()` - for ARDs with descriptive statistics for continuous, categorical and dichotomous variables.
+-   `tbl_ard_wide_summary()` - for ARD statistics represented in a wide table format - in separate columns.
+
+### Building a basic demographics table
+
+In this example, we will build a simple demographics and baseline characteristics table as outlined in the FDA Standard Safety Tables Guidelines.
+
+This table has three types of data summaries: a continuous variable summary for AGE, a categorical variable summary for AGEGR1, RACE, and ETHNIC and a dichotomous variable summary for SEX.
+
+```{r}
+library(gtsummary)
+library(cards)
+
+# summarizing both continuous and categorical variables, consolidate using ard_stack call
+ard_stack(
+  data = cards::ADSL,
+  .by = ARM,
+  ard_categorical(variables = c("AGEGR1", "RACE", "SEX", "ETHNIC")),
+  ard_continuous(variables = "AGE"),
+  .attributes = TRUE,
+  .missing = TRUE,
+  .total_n = TRUE,
+  .overall = TRUE # generate ard with overall column included
+  ) |> # pass to tbl_ard_summary to generate a table.
+  tbl_ard_summary(by = ARM, overall = TRUE) # pass ard to table building function with the overall argument.
+
+```
+
+Notice this ARD first approach builds an equivalent table to one built with `tbl_summary()`
+
+```{r}
+
+cards::ADSL |>
+  select(c("AGEGR1", "RACE", "SEX", "ETHNIC", "AGE", "ARM")) |>
+  tbl_summary(by = ARM) |>
+  add_overall() 
+
+```
+The ARD for this table can be extracted using the `gather_ard()` function.
+
+### Building a disposition table
+Here, we use tbl_ard_wide_summary to build a Pooled disposition table. 
```

Comment:
Can you also include the vanilla `tbl_summary()` code along with `gather_ard()` ?